### PR TITLE
feat: bump up cosmwasm from v1.0.0 to v1.1.9

### PR DESCRIPTION
**Pull request by zemyblue**
# Description

* bump up cosmwasm from v1.0.0 to v1.1.9
  * the changes is [cosmwasm v1.0.0...v1.1.9](https://github.com/CosmWasm/cosmwasm/compare/v1.0.0...v1.1.9)

### Add
* cosmwasm-std: Implement PartialEq for Binary and u8 arrays.
* cosmwasm-std: Implement BankQuery::Supply to allow querying the total supply of a native token.
* cosmwasm-std: Add QuerierWrapper::query_wasm_contract_info
* cosmwasm-check: This is a new binary package that allows running various CosmWasm compatibility checks on compiled .wasm files. 

### Changes
* Upgrade Wasmer from 2.2.1 to 2.3.0 
* all: Bump a few dependency versions to make the codebase compile with -Zminimal-versions
* cosmwasm-std: Enable the abort feature by default. This provides more helpful panic messages via a custom panic handler.
* cosmwasm-crypto: Bumped k256 0.10.4 -> 0.11 and digest 0.9 -> 0.10
* cosmwasm-profiler: Package was removed
* cosmwasm-schema: schema generation is now locked to produce strictly draft-07 schemas

### Fixed
* cosmwasm-schema: The QueryResponses derive macro now supports QueryMsgs with generics. 

## Types of changes

- [x] Bug fix (changes which fixes an issue)
- [x] New feature (changes which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] ETC (build, ci, docs, perf, refactor, style, test)

## Checklist

- [x] I followed the [contributing guidelines](https://github.com/line/cosmwasm/blob/main/CONTRIBUTING.md).
- [x] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] The PR title and commits are followed [conventional commit form](https://www.conventionalcommits.org/en/v1.0.0).

### CODE REVIEW

**Comment by zemyblue** [comment]
@shiki-tak , @loloicci , is there any problem if I change the version to 1.1.9-0.7.0?

File: packages/std/schema/query_request.json [hunk 1]
```diff
@@ -42,27 +42,6 @@
...
-              ],
-              "properties": {
-                "denom": {
-                  "type": "string"
-                }
-              }
-            }
-          },
-          "additionalProperties": false
-        },
```
Comment (zemyblue) [inline packages/std/schema/query_request.json]: We need to check why this change should be removed for passing ci.

**Review by loloicci — COMMENTED** [review]
CHANGELOG is auto-updated when the version is updated in this repository. So Unreleased section is not needed.

**Review by loloicci — APPROVED** [review]

**Review by da1suk8 — COMMENTED** [review]
The `.circleci/config.yml` is empty, is there a reason why it has been added? 

The rest is ok.

**Review by shiki-tak — APPROVED** [review]
LGTM except for comments in the document

File: IBC.md [hunk 2]
```diff
@@ -0,0 +1,580 @@
+# IBC interfaces for CosmWasm contracts
```
[the remark this answers is not in the document]
Comment (zemyblue) [reply IBC.md:1]: OK. I'll remove it. Thank you.

**Comment by zemyblue** [comment]
> The `.circleci/config.yml` is empty, is there a reason why it has been added?
> 
> The rest is ok.

Oh, I did remove it. But I think it comes alive again when merging. I'll remove again. Thank  you.

**Review by da1suk8 — APPROVED** [review]

**Pull request merged**